packages/xo-web/src/common/intl/messages.js

```diff
@@ -1120,6 +1120,7 @@ const messages = {
   noIpv4Record: 'No IPv4 record',
   noIpRecord: 'No IP record',
   started: 'Started {ago}',
+  by: 'By',
```

Comment:
Translation should not be done by concatenating pieces of text, use a variable instead.